resources/build.js

```diff
@@ -0,0 +1,45 @@
+var execSync = require('child_process').execSync;
+var fs = require('fs');
+var rimraf = require('rimraf');
+var isWindows = /^win/.test(process.platform)
+
+var out = function (command) {
+  execSync(command, {stdio:'inherit'});
+};
+
+console.log('"Running node ./resources/build.js"');
+
+if (!isWindows) {
+  out('set -e'); // exit on console error
```

Comment:
I don't think this will do anything, will it?

`out` will fork a child process that lives for the duration of the `set` command, then exit, so the attempt to change the setting will have no subsequent effect.

I think we can delete this.